dev-php/fedora-autoloader/fedora-autoloader-0.2.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Fedora's Static PSR-4, PSR-0, and classmap autoloader"
+HOMEPAGE="https://github.com/php-fedora/autoloader"
+
+# here we use autoloadder as it is the upstream name
```

Comment:
You've got typo. Besides, I don't think the comment is really necessary. It's kinda obvious that SRC_URI uses upstream name, and having an explicit comment saying that makes me wonder it has some extra meaning — i.e. in the end, it causes only more confusion.

Is there any reason why you don't install it as `dev-php/autoloader` btw? You seem to be inconsistenly using bare package name for some packages, and vendor+package for some.